docker-compose.yml

```diff
@@ -9,8 +9,12 @@ services:
     volumes:
       - .:/usr/windbreaker-hooks
       - /usr/windbreaker-hooks/node_modules
+      # uncomment to link and/or replace first path if service utils are located elsewhere
+      # - ../windbreaker-service-util:/usr/windbreaker-hooks/node_modules/windbreaker-service-util
     environment:
       SERVICE_ENVIRONMENT: 'docker'
+    ports:
+      - 8080:8080
```

Comment:
Since this service/config is only run locally, it seems fine-ish to bind to a static port. It certainly makes it easier to test end-to-end w/ github pointing at ngrok. I can remove or comment out if others would prefer not to see it though.